src/Microcks.Testcontainers/Microcks.Testcontainers.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
I saw that .NET 9.0 is now around. Is it supported?

Reply:
Hello, as .NET7.0, .NET 9.0 is STS application. 

If your application is targeting .NET9.0, you should be able to use the package without major issues.

Few points to consider: 
**Backward compatibility: .**NET 9.0 (and normaly all next version) is designed to be backward compatible with earlier versions of .NET. 

**Fallback:** if the package includes a NET8.0 version, and your application is targeting .NET 9.0 will likely use this version.

**Specific Features:** Ensure that the features you need are available in the version of the package you are using. 
For exemple, it's possible to have a specific feature with `#if NET8_0` in code, I think, it's better for us to use `#if NET8_0_OR_GREATER` if the scenarii is neccessary tomorrow.